internal/users/cache/testdata/database_with_unknown_bucket.db.yaml

```diff
@@ -2,3 +2,15 @@ UserByID:
   "1111": '{"Name":"user1","UID":1111,"GID":11111,"Gecos":"User1 gecos\nOn multiple lines","Dir":"/home/user1","Shell":"/bin/bash","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"AAAAATIME"}'
 Unknown bucket:
   "somekey": "some value"
+GroupByID:
```

Comment:
I don’t think that file should be updated. The test using it wants to showcase that we create any missing buckets (and delete unknown ones). Maybe the fixture file should be change, but we should keep the tests doing both (here by creating all buckets, we will only delete unknown)

Reply:
oops, you're right. This change probably happened when my brain turned auto-pilot mode here.

Reply:
You introduce an empty line though in the diff while all the other files don’t have one. Let’s align those?